common/transport/src/main/proto/grpc/block_master.proto

```diff
@@ -69,6 +69,7 @@ message WorkerInfo {
   map<string, int64> capacityBytesOnTiers = 8;
   map<string, int64> usedBytesOnTiers = 9;
   optional BuildVersion buildVersion = 10;
```

Comment:
One more question, is the buildVersion used somewhere?

Reply:
I don't know